datacats/cli/main.py

```diff
@@ -81,7 +83,7 @@ def main():
         if help_:
             args = ['--help']
         return docopt(__doc__, args,
-            version=pkg_resources.require("datacats")[0].version)
+                      version=pkg_resources.require("datacats")[0].version)
```

Comment:
this is my one quibble with pep8: don't make me align to opening parens. this creates all sorts of noise when the first line changes and the rest need to be re-indented.